packages/admin/src/common/toolbar/Toolbar.styles.tsx

```diff
@@ -0,0 +1,62 @@
+import { Theme } from "@material-ui/core/styles";
+import { makeStyles } from "@material-ui/styles";
+
+import { useComponentThemeProps } from "../../mui/useComponentThemeProps";
+
+export interface ToolbarThemeProps {
+    elevation?: number;
+}
+
+export type CometAdminToolbarClassKeys = "root" | "muiToolbar" | "mainContentContainer" | "actionContainer";
+
+export const useStyles = makeStyles<Theme, { headerHeight: number }, CometAdminToolbarClassKeys>(
+    () => ({
+        root: {
+            position: "sticky",
+            display: "flex",
+            flexDirection: "column",
+            justifyContent: "center",
+            top: (props) => props.headerHeight,
+            padding: 0,
+            "& [class*='MuiButton']": {
+                textTransform: "none",
+            },
```

Comment:
I think this should be set for `MuiButton` globally, not just in the toolbar.

Reply:
especially it should be possible to override using theme.

Reply:
@jamesricky yes we should definitly do this with the mui button in general. I think i know that you already have soon a MR in place to do this? I will remove this special styling and wait for it.

@nsams of course this can be easily be overridden in @comet/theme or any other theme

e.g.:
<img width="1672" alt="Screenshot 2021-05-06 at 08 29 36" src="https://user-images.githubusercontent.com/6098356/117251629-3a5bc000-ae45-11eb-847f-5c9d40877a7d.png">